pkg/cvo/sync_worker.go

```diff
@@ -474,6 +480,8 @@ func (w *SyncWorker) syncOnce(ctx context.Context, work *SyncWork, maxWorkers in
 	validPayload := w.payload
 	if validPayload == nil || !equalUpdate(configv1.Update{Image: validPayload.ReleaseImage}, configv1.Update{Image: update.Image}) {
 		klog.V(4).Infof("Loading payload")
+		cvoObjectRef := &corev1.ObjectReference{APIVersion: "config.openshift.io/v1", Kind: "ClusterVersion", Name: "version"}
+		w.eventRecorder.Eventf(cvoObjectRef, corev1.EventTypeNormal, "RetrievePayload", "retrieving payload version=%q image=%q", update.Version, update.Image)
```

Comment:
it would be nice to have the version and image as structured data on the event.  Is that possible, or are flat strings all we have available?

Reply:
> it would be nice to have the version and image as structured data on the event. Is that possible, or are flat strings all we have available?

To my knowledge, flat strings are what you have.

Reply:
Technically we could do that as annotation, but let's not for now.